src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -451,6 +483,11 @@ def schedule_unit(self, unit):
                 # try next node
                 continue
 
+            # - PRRTE related - start -
+            if node_dvm_id is not None and unit_dvm_id is None:
```

Comment:
Since IDs never can be empty, this (and elsewhere) could be:
```py
if node_dvm_id and not unit_dvm_id:
```
?

Reply:
tricky, considering that the value could be `0`, that's why I forced to check everywhere that `None` is a not-set value, otherwise value is set

Reply:
Ah, right  - you use integers...  Sorry, I hade `xyz,000` type IDs in mind which we use otherwise...